apps/managedidentity/managedidentity.go

```diff
@@ -192,14 +195,34 @@ type AcquireTokenOption func(o *AcquireTokenOptions)
 // Use this option when Azure AD returned a claims challenge for a prior request. The argument must be decoded.
 func WithClaims(claims string) AcquireTokenOption {
 	return func(o *AcquireTokenOptions) {
-		o.claims = claims
+		if claims != "" {
+			o.claims = claims
+		}
+	}
+}
+
+// WithClientCapabilities sets the client capabilities to be used in the request.
+// This is used to enable specific features or behaviors in the token request.
+// The capabilities are passed as a slice of strings, and empty strings are filtered out.
```

Comment:
The docs should point to CAE https://learn.microsoft.com/en-us/entra/identity/conditional-access/concept-continuous-access-evaluation